.github/scripts/lib/emit-pipeline-metrics-v4-core.ps1

```
@@ -36,6 +36,48 @@
+    )
+
+    try {
+        $viewed = & $GhCliPath repo view --json nameWithOwner --jq '.nameWithOwner' 2>$null
+        if ($LASTEXITCODE -eq 0 -and -not [string]::IsNullOrWhiteSpace($viewed)) {
+            $candidate = $viewed.Trim()
+            if ($candidate -notmatch '^[A-Za-z0-9_.\-]+/[A-Za-z0-9_.\-]+$') { return $null }
+            return $candidate
+        }
+    } catch {
+        # fall through to git-remote parse
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)